en/micro-integrator/docs/references/connectors/kafka-connector/kafka-inbound-endpoint-example.md

```diff
@@ -53,35 +59,43 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
          </log>
       </sequence>
    ```
+## Exporting Integration Logic as a CApp
 
-2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
 
-3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="/assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
 
-    ```
-    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
-    ```
+2. Select an **Export Destination** where you want to save the .car file. 
 
-{!references/connectors/exporting-artifacts.md !}
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided in the previous step.   
 
 ## Deployment
-Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
 
-{!references/connectors/deploy-capp.md!}
+1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `Kafka`. Click on `Kafka Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. Copy this .jar file into  <PRODUCT-HOME>/lib folder. 
+
+2. Copy the exported carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder. 
+
+3. [Start the WSO2 EI server](https://docs.wso2.com/display/EI660/Running+the+Product).
```

Comment:
Let's use an EI 7.0.0 docs link here.

Reply:
Added https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/#start-the-micro-integrator